### PR TITLE
Enable render cache by default

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

**Comment by e0ipso** [comment]
@amitaibu tests are failing.

File: tests/RestfulEntityAndPropertyAccessTestCase.test [hunk 1]
```diff
@@ -211,6 +211,12 @@ class RestfulEntityAndPropertyAccessTestCase extends DrupalWebTestCase {
     $public_fields['label']['access_callbacks'][] = array($this, 'publicFieldAccessFalse');
     $handler->setPublicFields($public_fields);
 
+    // Clear the cache.
+    $handler->clearRenderedCache(array(
```
Comment (amitaibu) [inline tests/RestfulEntityAndPropertyAccessTestCase.test:215]: @mateu-aguilo-bosch maybe as a follow up we should add a `$handler->clearCache()` (to clear all handler's cache), and `$handler->clearCacheByResource($entity)` ?

**Comment by amitaibu** [comment]
@mateu-aguilo-bosch ready for review.

File: tests/RestfulEntityAndPropertyAccessTestCase.test (hunk 1 above)
Comment (e0ipso) [inline tests/RestfulEntityAndPropertyAccessTestCase.test:215]: Added https://github.com/Gizra/restful/issues/289

**Comment by e0ipso** [comment]
Merged.

Thanks!!

**Pull request closed without merging**